ExpenseMaster.BusinessLogic/Implementations/IncomeService.cs

```diff
@@ -28,7 +28,11 @@ public async Task<IEnumerable<IncomeItemDto>> GetAllIncomes()
         public async Task<IncomeItemDto> GetIncomeById(int id)
         {
             var result = await _repositoryWrapper.Income.FindByConditionAsync(x => x.Id == id);
-            var income = await result.FirstOrDefaultAsync();
+            var income = result.FirstOrDefault();
```

Comment:
Why do you change it? It is interesting for me) (p.s. you do not have to fix it)